frontend/src/component/project/ProjectApplications/ProjectApplications.tsx

```diff
@@ -0,0 +1,204 @@
+        setTableState({ query });
+    };
+
+    const bodyLoadingRef = useLoading(loading);
+
+    const { offset, limit, query, sortBy, sortOrder, ...filterState } =
+        tableState;
+
+    const columns = useMemo(
+        () => [
+            columnHelper.accessor('name', {
+                header: 'Name',
+                cell: ({ row }) => (
+                    <LinkCell
+                        title={row.original.name}
+                        to={`/applications/${row.original.name}`}
+                    />
+                ),
+                meta: {
+                    width: '25%',
+                },
+            }),
+            columnHelper.accessor('environments', {
+                header: 'Environments',
+                cell: ({ row }) => (
+                    <StringArrayCell
+                        row={row.original}
+                        field={'environments'}
+                        singularLabel={'environment'}
+                    />
+                ),
+                enableSorting: false,
+                meta: {
+                    width: '25%',
+                },
+            }),
+            columnHelper.accessor('instances', {
+                header: 'Instances',
+                cell: ({ row }) => (
+                    <StringArrayCell
+                        row={row.original}
+                        field={'instances'}
+                        singularLabel={'instance'}
+                    />
+                ),
+                enableSorting: false,
+                meta: {
+                    width: '25%',
+                },
+            }),
+            columnHelper.accessor('sdks', {
+                header: 'SDK',
+                cell: SdkCell,
+                enableSorting: false,
+                meta: {
+                    width: '25%',
+                },
+            }),
+        ],
+        [],
+    );
+
+    const table = useReactTable(
+        withTableState(tableState, setTableState, {
+            columns,
+            data: applications,
+        }),
+    );
+
+    const rows = table.getRowModel().rows;
+
+    return (
+        <PageContent
+            disableLoading={true}
+            bodyClass='no-padding'
+            header={
+                <PageHeader
+                    title='Project applications'
+                    actions={
+                        <>
+                            <ConditionallyRender
+                                condition={!isSmallScreen}
+                                show={
+                                    <>
+                                        <Search
+                                            placeholder='Search'
+                                            expandable
+                                            initialValue={query || ''}
+                                            onChange={setSearchValue}
+                                            id='globalFeatureToggles'
```

Comment:
Is this the right search id? I'm not familiar with the global feature toggle search, but it feels like a different feature from this.

Reply:
Good catch